src/manage_config_discovery.c

```diff
@@ -724,7 +721,6 @@ make_config_discovery (char *const uuid, char *const selector_name)
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.800029", "General");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.800257", "General");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.800108", "General");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.100353", "General");
```

Comment:
https://secinfo.greenbone.net/omp?cmd=get_info&info_type=nvt&info_id=1.3.6.1.4.1.25623.1.0.100353&token=guest

-> Deprecated and also a Policy-Check -> Not relevant for a discovery